docs/tutorial/acquiring_data/fido.rst

```diff
@@ -218,7 +236,9 @@ This means that the results of a call to search can contain many sets of records
 The results of a search are represented in a `~sunpy.net.fido_factory.UnifiedResponse` object, which provides access to all the response tables and allows some operations to be performed on all the results at once.
 `~sunpy.net.fido_factory.UnifiedResponse` acts both like a two dimensional array, where the first dimension is the response index and the second index is the row index, and a dictionary where you can index the responses by the name of the client.
 
-For example, the following code returns a response containing LYRA data from the `~sunpy.net.dataretriever.LYRAClient`, and EVE data from the `~sunpy.net.vso.VSOClient`::
+For example, the following code returns a response containing LYRA data from the `~sunpy.net.dataretriever.LYRAClient`, and EVE data from the `~sunpy.net.vso.VSOClient`:
+
+.. code-block:: python
```

Comment:
Why?

Reply:
The code-block directive is the proper way to display code blocks using sphinx.
It allows us to tweak settings if we need to, it allows us to pin the language without having sphinx try to guess it. 
It highlights the code in my editor whereas the `::` does not.
It is neater in the source rst file.
NASA has mandated it as part of the ROSES 2023 call.

I plan to do this change to every rst file with code.